### ISSUE TITLE
ERROR: Condition "font.is_null()" is true at: _shape (scene/gui/line_edit.cpp:2202) on project open

### ISSUE THREAD

**Issue by hsandt** [godotengine/godot#67388]
### Godot version

v4.0.beta.custom_build [99bc4905c]

### System information

Linux Ubuntu 20.04 with Unity desktop

### Issue description

On project launch (new project creation or open project), I have an error:

```
ERROR: Condition "font.is_null()" is true.
   at: _shape (scene/gui/line_edit.cpp:2202)
```

The error occurs in v4.0.beta.custom_build [99bc4905c], but doesn't occur on v3.4.1.stable.official [aa1b95889] (I didn't test v3.5.1 stable).

### Steps to reproduce

Open Godot and either create a new project or open an existing one.

**Comment by bruvzg** [comment]
Seems like `EventListenerLineEdit` subclass is setting placeholder in the constructor, before font is set or theme is initialized.

**Comment by akien-mga** [comment]
Pinning the issue as users will encounter it in 4.0 beta 3 (it's fixed in `master` and thus will be fixed in 4.0 beta 4).

**Comment by EricEzaM** [comment]
I did not realise it was my changes that caused this! Apologies.
